### PR TITLE
Add new module: pagerduty_alert

### PR DESCRIPTION
**Pull request by ApsOps**
- trigger, acknowledge or resolve pagerduty incidents

### CODE REVIEW

**Comment by bcoca** [comment]
I would move to monitoring/ dir as that is where we have modules that deal with monitoring services

File: monitoring/pagerduty_alert.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    - "Amanpreet Singh (@aps-sids)"
+requirements:
+    - PagerDuty API access
+options:
+    service_key:
+        description:
+            - The GUID of one of your "Generic API" services.
+            - This is the "service key" listed on a Generic API's service detail page.
+        required: true
+    event_type:
```
Comment (bcoca) [inline monitoring/pagerduty_alert.py]: A more imperative language here might be more inline with anisble modules:

state: triggered, acknowledged, resolved

also check if the incident is already in that state to report on changed correctly

**Comment by aps-sids** [comment]
Thanks @bcoca . Did that.

File: monitoring/pagerduty_alert.py (hunk 1 above)
Comment (ApsOps) [inline monitoring/pagerduty_alert.py]: Okay, makes sense.

**Comment by ApsOps** [comment]
@bcoca Did the changes as per your suggestions. Please take a look.

**Pull request merged**